app/[locale]/job/add/page.test.tsx

```diff
@@ -38,20 +38,4 @@ describe('Add Income To Ledger Page', async () => {
             asPath: "/job/add"
         })
     })
-
-    it('Navigates when fields are filled in', async () => {           
```

Comment:
this i pulled out to because it's a routing test, essentially, and i'd like to move all routing tests to its own section under `test/`